sv_SE/LC_MESSAGES/playlist.po

```diff
@@ -4,136 +4,82 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 
 msgid "ADDED_TO_NEW_PLAYLIST"
-msgstr "Added to new Playlist"
+msgstr "Tillagd i ny spellista"
 
 msgid "ADDED_TO_PLAYLIST"
-msgstr "Added to Playlist"
-
-msgid "BACK_TO_CREATORS_ROOM"
-msgstr "Back to Creator's Room"
+msgstr "Tillagd i spellista"
 
 msgid "BACK_TO_FLIPNOTE"
-msgstr "Back to Flipnote"
+msgstr "Tillbaka till Flipnote"
 
 msgid "CREATOR"
-msgstr "Creator:"
```

Comment:
String missing for `CREATOR`, ID/string missing for `DELETE_PLAYLIST`, `DELETE_PLAYLIST_CONFIRM_1`, `DELETE_PLAYLIST_CONFIRM_2`, `DELETE_PLAYLIST_CONFIRM_NO`, missing ID for `DELETE_PLAYLIST_CONFIRM_YES`